src/redux/reducers/cats.js

```diff
@@ -8,6 +8,7 @@ const initialState = {
     { name: 'Graham', description: 'Ragdoll', img: 'https://www.pets4homes.co.uk/images/breeds/28/large/28b9defa5a9609f4d99b38e1c67424f3.jpg' },
   ],
   cat: { name: 'Marley', description: 'Ocicat', img: 'https://www.pets4homes.co.uk/images/breeds/116/large/91e94b7e726f19ff3973de575ec1520c.jpg' },
+  adoptedCats: [],
```

Comment:
Great, so we now have an array in state that is going to hold our adopted cats! Now we need to figure out how to put new cats in it.